app/models/apple/show.rb

```diff
@@ -31,6 +31,15 @@ def self.get_show(api, show_id)
       api.unwrap_response(resp)
     end
 
+    def self.for_podcast(podcast)
```

Comment:
Not a big deal,  but the pattern up to now has been to build derived objects _from_ some other object. so `from_podcast(podcast)` here for the signature?

Reply:
for sure!